packages/aws-cdk/test/integ/run-against-dist.bash

```diff
@@ -22,17 +22,53 @@ function serve_npm_packages() {
     return
   fi
 
-  tarballs=$dist_root/js/*.tgz
+  if [ ! -z ${USE_PUBLISHED_FRAMEWORK_VERSION:-} ]; then
```

Comment:
I feel this should be exposed as `run-against-release-with-dist-cli` script instead of a switch to this one with a huge branch in it?

Reply:
Also, quote inside the `if`

Reply:
> I feel this should be exposed as run-against-release-with-dist-cli 

Not sure this can be easily achieved without some non trivial duplication. There are things in `run-against-dist` that I haven't investigated their need or usage. Mind if we push this to a later date as well?